.yamato/package-tests.yml

```diff
@@ -27,4 +27,31 @@ test_{{project.name}}_{{  package.name }}_{{ editor }}_{{ platform.name }}:
 {% endfor -%}
 {% endfor -%}
 {% endfor -%}
+{% endfor -%}
+
+# Test minimal project with different versions of dependencies
+{% for project in projects -%}
+{% if project.name == "minimalproject" -%}
+{% for dependency in dependencies -%}
+{% for editor in dependency.test_editors -%}
+{% for platform in test_platforms -%}
+test_compatibility_{{project.name}}_{{ project.packages.first.name }}_with_{{ dependency.name }}@{{ dependency.version }}_{{ editor }}_{{ platform.name }}:
+  name : {{ project.name }} - {{ project.packages.first.name }} with {{ dependency.name }}@{{ dependency.version }} - {{ editor }} on {{ platform.name }}
+  agent:
+    type: {{ platform.type }}
+    image: {{ platform.image }}
+    flavor: {{ platform.flavor}}
+  commands:
+    - npm install upm-ci-utils@stable -g --registry https://artifactory.prd.cds.internal.unity3d.com/artifactory/api/npm/upm-npm
+    - {% if platform.name == "ubuntu" %}DISPLAY=:0 {% endif %}upm-ci project test -u {{ editor }} --type project-tests --project-path {{ project.name }} --package-filter {{ package.name }}
+  artifacts:
+    logs:
+      paths:
+        - "upm-ci~/test-results/**/*"
+  dependencies:
+    - .yamato/project-pack.yml#pack_{{ project.name }}_{{ dependency.name }}@{{ dependency.version }}
```

Comment:
This entire config is pretty much the same as the one above, except its dependency is a `Project Packed with UTP 2.0`